### DISCUSSION TITLE
Marking tests as skipped

### DISCUSSION THREAD

**Discussion by ousid** [pestphp/pest#621] [Ideas]
Is there any way to mark the all the tests as `skipped`  in a single file, or is there any possibility to be added in `v2`

Something like

```php
/** @skipped */

it('...', ...);

...

```
or

```php
markAsSkipped();

....
```

### Use cases
Sometimes, you may delay some failed tests till some point (in case you want to run the whole suit)....

I will be happy to contribute, or at least give it a try!

@nunomaduro

**Comment by nunomaduro** [comment]
Try `beforeEach()->skip()`.

**Reply by ousid** [reply]
Interesting! I didn't think of this.

Thanks for the hint, nuno!

Edit: I have added a [PR](https://github.com/pestphp/docs/pull/137) into the docs, to document the piece of skipping all the tests. It may be helpful for folks who want to know.
